### ISSUE TITLE
hackbox-demo uses a function that does not exist in hackbox-client v0.0.3

### ISSUE THREAD

**Issue by jackdaus** [tomalama/hackbox#19]
Hello, 

I setup and ran the hackbox-demo package, but I ran into a bug. On the host screen (i.e., http://localhost:3000/host), when we click the Start Game button, we get the following error message: 

TypeError: hackbox.startGame is not a function
onClickStartGame
C:/Git/hackbox/packages/hackbox-demo/src/client/components/Host.js:32
`  const onClickStartGame = () => {
    hackbox.startGame({roomId: room.id, gameType});
  };
`

It looks like the hackbos.startGame() function was added to the hackbox-client package during commit 1e85346e1933a708bf89f0bc777f12449e8ec398 (on 1/7/2020). But this change has not yet been pushed to the hackbox-client npm package. So when I run npm install hackbox-client, I am getting a version of this package that does not have this function. Since the hackbox-demo project imports hackbox-client from the node_modules, this new function does not exist for the hackbox-demo project. 

Maybe we can fix this issue by updating the npm package to v 0.0.4 that includes the changes from commit 1e85346e1933a708bf89f0bc777f12449e8ec398? :)

**Comment by Blak-Rabbit** [comment]
I can't even see any start button on the host page. Am I missing something? Thank you in advance for your help :)

**Comment by jackdaus** [comment]
I don't think this project is being maintained anymore. Looks like it was more of a prototype at the time.

**Comment by Blak-Rabbit** [comment]
Dang. Hoped it could work with what was here. Thanks!
